main.js

```diff
@@ -1,18 +1,123 @@
 var titleField = document.querySelector('#submitted-title');
 var bodyField = document.querySelector('#submitted-body');
-var saveButton = document.querySelector('#save-button');
+var saveButton = document.querySelector('.save-button');
+var ideaGrid = document.querySelector('#card-display');
```

Comment:
Thinking some of these should be classes as opposed to id's.

Reply:
I'll add this to the running list of possible refactors! Great suggestion.